survey/tests/testdump.json

```diff
@@ -94,14 +94,26 @@
  "model": "survey.survey",
  "pk": 4,
  "fields": {
-  "name": "Test surv\u00eby2",
```

Comment:
Actually, the Unicode in the name is part of the test :)

Reply:
Fixed!